Remove the sanity checking from the rpm autogen.sh.  This is required because
we may have slightly different, but yet compatible versions.  If we do end
up breaking things, we'll deal with it at that time.

Upstream-Status: Inappropriate [configuration]

Signed-off-by: Mark Hatle <mark.hatle@windriver.com>

Index: rpm-5.4.14/syck/autogen.sh
===================================================================
--- rpm-5.4.14.orig/syck/autogen.sh
+++ rpm-5.4.14/syck/autogen.sh
@@ -34,12 +34,6 @@ libtoolize () {
     eval $_libtoolize $_libtoolize_args
 }
 
-#   requirements sanity check
-[ "`automake   --version | head -1`" != "$AMV" ] && echo "$USAGE" # && exit 1
-[ "`autoconf   --version | head -1`" != "$ACV" ] && echo "$USAGE" # && exit 1
-[ "`libtoolize --version | head -1`" != "$LTV" ] && echo "$USAGE" # && exit 1
-[ "`gettextize --version | head -1 | sed -e 's;^.*/\\(gettextize\\);\\1;'`" != "$GTT" ] && echo "$USAGE" # && exit 1
-
 echo "---> generate files via GNU libtool (libtoolize)"
 libtoolize --quiet --copy --force --install
 echo "---> generate files via GNU autoconf (aclocal, autoheader)"
